### DISCUSSION TITLE
Help with website

### DISCUSSION THREAD

**Discussion by MitkoVtori** [community/community#34797] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I am trying to publish my website and for a test I paste only a small part of my code into the repo to check if it will work. 
The site exists but there I can only see the README.md file and link to the project or something. Why my code doesn't do anything there?
Also, my website has 5 pages, all of them are different chrome sites, and I use links so in any of this 5, you can go back.
![upper_part](https://user-images.githubusercontent.com/112943652/193408967-6c563e6b-6bb9-4e8a-bef4-863d075ac24a.png)
This image show a small part of it, if you click on certificates it should send you on the certificates page will it work when i send it, or I have to create a repo for each page?

**Comment by radar-blip** [comment, marked as answer]
You dont need a repo for each page, all your files should be in the same repo.
After that, its just a matter of getting all the links connected accordingly.

**Comment by jfullstackdev** [comment]
but that seems to be working fine on my end 

![image](https://user-images.githubusercontent.com/47092464/193434589-69ae66ed-b4cc-46a2-a9bb-c4aca58a8ae0.png)

**Comment by MitkoVtori** [comment]
Yeah, I find how to fix everything. Thank you anyways!

**Comment by ossanna16** [comment]
Great question @MitkoVtori! I'm glad you found our community 🙂
